scripts/scripts.js

```diff
@@ -207,8 +208,7 @@ async function loadPage() {
   await loadLazy(document);
   loadDelayed();
 }
+addPrefetch('preconnect', 'https://icicidirect-secure-worker.franklin-prod.workers.dev');
```

Comment:
@sdmcraft should we add this to end of the eager phase or start of lazy phase. 
We are not making any API call before eager phase.